src/panels/lovelace/editor/view-editor/hui-view-editor.ts

```diff
@@ -80,20 +82,32 @@ export class HuiViewEditor extends LitElement {
       ${configElementStyle}
       <div class="card-config">
         <paper-input
-          label="Title"
+          .label="${this.hass.localize(
+            "ui.panel.lovelace.editor.card.generic.title"
```

Comment:
Hmm this isn't a card, so we should have put this under `ui.panel.lovelace.editor.generic.` we will get the same with badges...

Reply:
Yeah thought the same thing but didn't want to refactor all the translations for editors as well and didn't want to make a 2nd set of the same thing. `ui.panel.lovelace.editor.generic.title` ideally, I think in the future